insomnihack-ctf-2016/.gitignore

```diff
@@ -0,0 +1,5 @@
+backdoor/smartdoor1/smartdoor1_2ed98791bd003533dc8b4b5ea79a6656
+backdoor/smartdoor2/smartdoor2_2ceb2e595d5f58fa55297e2e1bbc274b
+backdoor/smartdoor3/smartdoor3_6c931f4d5090f6a360bee7db3681c0d3
+hardware/iotcode/iotcode_8dfe573cdbccd541375408706d2c583c.zip
+shellcode/superpollute/qemu-sh4_74ca7af34c287f9d3f20d126bad26fe7.tgz
```

Comment:
@bcyrill so these file are greater than 10 MB so did you upload them to somewhere else (cloud) and added the link here ?